doc/nrf/glossary.rst

```diff
@@ -681,6 +681,9 @@ Glossary
    Simple Network Time Protocol (SNTP)
       A less complex implementation of :term:`Network Time Protocol (NTP)`, using the same protocol but without requiring the storage of state over extended periods of time.
 
+   Software-Enabled Access Point (SoftAP or AP)
+      A network device operating as an IEEE 802.11 access point using software with limited functionality and no connection to the wired network.
```

Comment:
Now, I am thinking, this contradicts a bit with the definition of SoftAP in the guide (... as a means of sharing an internet connection). Can we drop the last part?

```suggestion
      A network device operating as an IEEE 802.11 access point using software with limited functionality.
```